templates/frontend/pages/indexJournal.tpl

```diff
@@ -60,17 +60,19 @@
 								</article>
 						{/foreach}
 					</div>
-					{* Carousel controls *}
-					<div class="text-right">
-						<a href="#announcementsCarouselControls" class="btn" role="button" data-slide="prev">
-							<span aria-hidden="true">←</span>
-							<span class="sr-only">{translate key="help.next"}</span>
-						</a>
-						<a href="#announcementsCarouselControls" class="btn" role="button" data-slide="next">
-							<span aria-hidden="true">→</span>
-							<span class="sr-only">{translate key="help.previous"}</span>
-						</a>
-					</div>
+					{if $announcements|@count > 1}
```

Comment:
2 conditions should be met here. `$numAnnouncementsHomepage` should also be more than 1. This variable defines user input of how many announcements should be shown. `$announcements` array defines only how many announcements are published. So, both of them should be more than 1.

Reply:
Sorry, haven't noticed before. Can you also put a condition to break the foreach loop with announcements iteration, e.g. like on the default indexJournal template: https://github.com/pkp/ojs/blob/master/templates/frontend/pages/indexJournal.tpl#L44-L46
Thus, we ensure that only the number of announcements specified by the user is shown.

Reply:
fixed and force-pushed!